Extensions/shuffle+.js

```diff
@@ -131,9 +131,10 @@
      */
     const fetchPlaylist = async (uri) => {
         const res = await Spicetify.CosmosAsync.get(`sp://core-playlist/v1/playlist/${uri}/rows`, {
-            policy: { link: true },
+            policy: { link: true, playable: true },
         });
-        return res.rows.map((item) => item.link);
+        const available = res.rows.filter((track) => track.playable);
+        return available.map((item) => item.link);
```

Comment:
Don't really see the point in this.
```suggestion
        return res.rows.filter((track) => track.playable).map((item) => item.link);
```

Reply:
Me neither, just keeping to what I've seen on https://github.com/spicetify/spicetify-cli/blob/master/Extensions/shuffle+.js#L224-L225. Fixed.